streams/src/test/java/org/apache/kafka/streams/integration/AdjustStreamThreadCountTest.java

```diff
@@ -466,64 +468,9 @@ public void close() {
                 waitForTransitionFromRebalancingToRunning();
 
                 for (final String log : appender.getMessages()) {
-                    // after we replace the thread there should be two remaining threads with 5 bytes each
-                    if (log.endsWith("Adding StreamThread-3, there are now 3 threads with cache size/max buffer size values as 3/178956970 per thread.")) {
-                        return;
-                    }
-                }
-            }
-        }
-        fail();
-    }
-
-    @Test
-    public void shouldResizeMaxBufferAfterThreadReplacement() throws InterruptedException {
```

Comment:
This test seems to be identical to the one above except that it uses the default cache size and a custom input buffer size -- however both tests are still validating both sizes in the log message so this doesn't need to be a separate test, we can just cover both in the original test.

(The total time to run streams tests has been getting a bit out of hand lately, so we should have a concrete reason to split something out into a second test like this, especially the more  "heavy"  integration tests)